config/default.yml

```diff
@@ -88,30 +88,94 @@ linters:
     # These cops are incredibly noisy when it comes to HAML templates, so we
     # ignore them.
     ignored_cops:
-      - Lint/BlockAlignment
-      - Lint/EndAlignment
-      - Lint/Void
-      - Layout/AlignHash # renamed to Layout/HashAlignment in rubocop 0.77
-      - Layout/AlignParameters # renamed to Layout/ParameterAlignment in rubocop 0.77
-      - Layout/ArgumentAlignment
-      - Layout/CaseIndentation
-      - Layout/ElseAlignment
-      - Layout/EndOfLine
-      - Layout/HashAlignment
-      - Layout/IndentationWidth
-      - Layout/LineLength # renamed from Metrics/LineLength in rubocop 0.79.0
-      - Layout/ParameterAlignment
-      - Layout/TrailingBlankLines # renamed to Layout/TrailingEmptyLines in rubocop 0.77
-      - Layout/TrailingEmptyLines
-      - Layout/TrailingWhitespace
-      - Metrics/BlockLength
-      - Metrics/BlockNesting
-      - Metrics/LineLength
-      - Naming/FileName
-      - Style/FrozenStringLiteralComment
-      - Style/IfUnlessModifier
-      - Style/Next
-      - Style/WhileUntilModifier
+      # We allow the LineLength cop (by not having a Enabled: false for it),
+      # This way, other cops which rely on it can do so.
+      - <%= rubocop_version < '0.79' ? 'Metrics/LineLength' : 'Layout/LineLength' %>
+
+    sent_to_rubocop:
```

Comment:
Can we add a comment explaining why this is needed versus a separate `.rubocop.haml-lint.yml` file?

Might make more sense here, but `sent_to_rubocop` is an odd setting name.

Reply:
I have no problem extracting it to a file. Not being my gem, I tried not to change everything, and "config" for rubocop was already in that file (ignored lints), but I needed more granularity. If you prefer a dedicated file, it can be done.